Src/Plugins/Security/SEC101_054.AlibabaAccessKeyValidator.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+// From the Nuget package
+using Aliyun.Acs.Core;
+using Aliyun.Acs.Core.Exceptions;
+using Aliyun.Acs.Core.Profile;
+
+// From the DLL
+using Aliyun.Acs.Iot.Model.V20170420;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class AlibabaAccessKeyValidator : ValidatorBase
+    {
+        internal static AlibabaAccessKeyValidator Instance;
+
+        protected const string KeyNotFound = "InvalidAccessKeyId.NotFound";
```

Comment:
>protected [](start = 8, length = 9)

we could change to private

Reply:
Done.

---
In reply to: [591816241](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591816241) [](ancestors = 591816241)